app/templates/forms/form/instances.hbs

```diff
@@ -1,13 +1,36 @@
 {{page-title "Instances"}}
-<div class="au-c-body-container au-c-body-container--scroll">
-  <SemanticForms::InstanceTable
-    @sort={{this.sort}}
-    @page={{this.page}}
-    @size={{this.size}}
-    @filter={{this.filter}}
-    @search={{this.search}}
-    @formDefinition={{this.model.formDefinition}}
-    @onRemoveInstance={{this.onRemoveInstance}}
-    @editRoute="forms.form.instance"
-  />
-</div>
\ No newline at end of file
+
+<AuMainContainer as |main|>
+  <main.sidebar>
+    <div class="au-c-sidebar">
+      <div class="au-c-sidebar__header au-u-margin-top-small">
+        <AuHeading @level="3" @skin="3">Filters</AuHeading>
+      </div>
+      <div class="au-c-sidebar__content">
+        <InstanceTableConfiguration
```

Comment:
should this component not just be inside the InstanceTable on the side of the semantic-forms addon? That way this component will also be much more clean I think. You can just make it configurable with a parameter like `@configurableLabels` or something?